sageresearch-app-sdk/src/main/java/org/sagebionetworks/research/sageresearch/viewmodel/ScheduledRepositorySyncStateDao.kt

```diff
@@ -0,0 +1,38 @@
+package org.sagebionetworks.research.sageresearch.viewmodel;
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.SharedPreferences
+import android.support.annotation.VisibleForTesting
+import org.joda.time.DateTime
+import javax.inject.Inject
+
+open class ScheduledRepositorySyncStateDao @Inject constructor(context: Context) {

```

Comment:
In Kotlin, you can define multiple class inside a single file.  I'd suggest moving this below the ScheduleRepository code, that way it is all in one spot.